vendor/bat-native-ads/src/bat/ads/internal/account/transactions/transactions_unittest.cc

```diff
@@ -32,11 +34,13 @@ TEST_F(BatAdsTransactionsTest, Add) {
 
   transactions::GetForDateRange(
       DistantPast(), DistantFuture(),
-      [&expected_transactions](const bool success,
-                               const TransactionList& transactions) {
-        ASSERT_TRUE(success);
-        EXPECT_EQ(expected_transactions, transactions);
-      });
+      base::BindOnce(
+          [](TransactionList expected_transactions, const bool success,
+             const TransactionList& transactions) {
+            ASSERT_TRUE(success);
+            EXPECT_EQ(expected_transactions, transactions);
+          },
+          std::move(expected_transactions)));
```

Comment:
has no effect because expected_transactions is const. See other tests which have same issue.

Reply:
fixed